src/main/java/uk/gov/hmcts/cmc/claimstore/services/FormaliseResponseAcceptanceService.java

```diff
@@ -110,31 +109,40 @@ private Offer prepareOffer(Response response, PaymentIntention paymentIntention)
             case BY_SPECIFIED_DATE:
                 LocalDate completionDate = paymentIntention.getPaymentDate().orElseThrow(IllegalStateException::new);
                 builder.completionDate(completionDate);
-                String contentBySetDate = prepareOfferContentsBySetDate(response, builder, completionDate);
-                builder.content(contentBySetDate);
+                builder.content(
+                    prepareOfferContentsBySetDate(response, builder, completionDate, claimAmountTillDate)
+                );
                 break;
             case INSTALMENTS:
                 RepaymentPlan repaymentPlan = paymentIntention.getRepaymentPlan().orElseThrow(IllegalAccessError::new);
                 builder.completionDate(repaymentPlan.getCompletionDate());
-                builder.content(String.format(
-                    "%s will pay instalments of %s %s. The first instalment will be paid by %s.",
-                    response.getDefendant().getName(),
-                    formatMoney(repaymentPlan.getInstalmentAmount()),
-                    repaymentPlan.getPaymentSchedule().getDescription(),
-                    formatDate(repaymentPlan.getFirstPaymentDate())
-
-                ));
+                builder.content(
+                    prepareOfferContentForRepayment(response, repaymentPlan, claimAmountTillDate)
+                );
                 break;
             default:
                 throw new IllegalStateException("Invalid payment option " + paymentIntention.getPaymentOption());
         }
         return builder.build();
     }
 
+    private String prepareOfferContentForRepayment(Response response, RepaymentPlan repaymentPlan,
+                                                   BigDecimal claimAmountTillDate) {
+        return String.format(
+            "%s will repay %s in instalments of %s %s. The first instalment will be paid by %s.",
+            response.getDefendant().getName(),
+            response.getResponseType() == ResponseType.PART_ADMISSION
```

Comment:
It may be a good idea to extract those as variables to improve readability. Leave it up to you though 👍

Reply:
Also will this always be in english ?

Reply:
@ckalista good point. The frequency comes as "Every week". Now I have fixed it. so it wont be Upper case in mid of sentence.

Reply:
@srineelesh I think it will be in english for now. Will need to ask BA for internalisation.